MobileWallet/TariLib/Wrappers/Utils/TariSettings.swift

```diff
@@ -107,8 +106,16 @@ struct TariSettings {
     //Used for showing a little extra detail in the UI to help debugging
     let isDebug = true
     let maxMbLogsStorage: UInt64 = 5000 //5GB
+    let pushNotificationServer = "https://1c1c2bea.ngrok.io"
+    let expirePendingTransactionsAfter: TimeInterval = 60 //A minute
     #else
     let isDebug = false
     let maxMbLogsStorage: UInt64 = 500 //500MB
+    let pushNotificationServer = "https://push.tari.com"
+    let expirePendingTransactionsAfter: TimeInterval = 60 * 60 * 24 * 3 //3 days
```

Comment:
I prefer suffixing var names like this with the time unit abbrev (secs, ms, etc), so I know later when I'm reading if it was seconds, minutes, etc.

Reply:
At first I had it as hours and an Int but then I figured if the type was TimeInterval it's implied the var is in seconds?

Reply:
I don't know if my point is valid but, imagine I'm reading it textually as I do here without the help of Xcode to see the type.

Reply:
It's a minor thing nevermind.